hawkular-inventory-parent/hawkular-inventory-service/src/main/java/org/hawkular/inventory/handlers/InventoryHandlers.java

```diff
@@ -183,7 +183,12 @@ public Response deleteResourceType(@PathParam("typeId") final String typeId) {
     public Response status(@Context ServletContext servletContext) {
         final Map<String, String> status = new HashMap<>();
         status.putAll(manifestUtil.getFrom());
-        status.put("status", "UP");
-        return Response.ok(status).build();
+        if (inventoryService.isRunning()) {
+            status.put("status", "UP");
+            return Response.ok(status).build();
+        } else {
+            status.put("status", "DOWN");
+            return Response.status(Response.Status.INTERNAL_SERVER_ERROR).entity(status).build();
```

Comment:
503 Service Unavailable would be more appropriate I think

Reply:
+1 oh yes, I was thinking on that but I left the 500 alone.
thx for reviewing it